edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -997,25 +949,25 @@ If your Delegate community has shared equipment, use some time after the competi
 - **Score taking errors**: Always double-check scorecards after they are entered. This can be done during or after the competition is finished, but always **before **submitting the results (help from reliable organizers or volunteers can be enrolled). When doing this, remember to check for missing signatures from the judge or competitor and verify that extra attempts were entered at their correct positions in the average, and that the ambiguous times are always entered in the worst reasonable interpretation according to [Guideline A7c+](https://www.worldcubeassociation.org/regulations/guidelines.html#A7c+). It is good practice to have a different person from the initial data enterer perform the double-checking, in case handwriting is interpreted differently.
 
-Also be aware of potential scoretaking errors. Scoretaking software like WCA Live will automatically check for results that exceeded the time limit or did not pass the cutoff, as long as WCA Live is synchronized with the times set in the competition’s edit page on the WCA website. However, if you are not using WCA Live, or if your scoretaking software does not automatically check that the results comply with the WCA Regulations, check that all results comply with the stated cutoffs, time limits, and cumulative time limits that were in place for that round of the competition.
+Also be aware of potential scoretaking errors. Scoretaking software like WCA Live will automatically check for results that exceeded the time limit or did not pass the cutoff, as long as WCA Live is synchronized with the WCA website. However, if you are not using WCA Live, or if your scoretaking software does not automatically check that the results comply with the WCA Regulations, check that all results comply with the stated cutoffs, time limits, and cumulative time limits that were in place for that round of the competition.
 
-- **Cutoffs**: If people do not meet the cutoff for a combined round, their further results have to be blank and they will have no average result. Blank results are different from DNF or DNS results, and are recorded as "no result".
-- **Time limits**: Times that exceeded the time limits must be changed to DNF.
-- **Cumulative time limits**: Make sure that the round format is correct (i.e. not using a cutoff), and that all times for a round with a cumulative time limit do not sum up to exceed the limit.
+- **Cutoffs**: if people do not meet the cutoff for a combined round, their further results have to be blank and they will have no average result. Blank results are different from DNF or DNS results, and are recorded as "no result".
+- **Time limits**: attempts that exceeded the time limits must be changed to DNF.
+- **Cumulative time limits**: if a round has a cumulative time limit, make sure that the round format is correct (i.e. not using a cutoff), and that the total of all attempts from one result does not exceed the limit.
```

Comment:
I'm not sure "one result" is correct.  It also gets confusing with Cumulative time limits shared between rounds and I feel this becomes less applicable.

Reply:
What wording would you recommend? Could also just go back to the old wording and say "round(s)".

Reply:
I think cumulative time limits between events complicate wording significantly.  I'd probably encourage your suggested change of going back to the old wording and saying "round(s)" and probably revisit it in the future.

Reply:
How about now?

Reply:
At least when I read "one result", it implies that it's referring to one solve/one attempt, which would not be the case?

Reply:
Oh, the diff in this review is just outdated now. It now says: "**Cumulative time limits**: if a round has a cumulative time limit, make sure that the round format is correct (i.e. not using a cutoff), and that the total of all attempts from the same round (or multiple rounds, if the cumulative limit is shared) does not exceed the limit."

See [diff](https://github.com/thewca/wca-documents/pull/432/files#diff-17dcd7e4aece71f465ce95ce6a0d1606769afe3fd83e6c1c24252e222dfb7d0d)

Reply:
Great!